crates/nu-parser/src/lex/tests.rs

```diff
@@ -131,6 +131,35 @@ def e [] {echo hi}
         );
     }
 
+    #[test]
+    fn lex_multi_comments() {
```

Comment:
I added this test when trying to figure out was the issue was and I figured it didn't hurt to keep it in. If you think it adds maintenance burden, I can remove it ofc.